src/pages/index.js

```diff
@@ -21,8 +28,8 @@ const IndexPage = () => (
         <p>Software Development and Delivery</p>
       </li>
     </ul>
-    <h2>Areas of Expertise</h2>
-    <ul>
+    <h3>We master following areas of Expertise</h3>
```

Comment:
Bad english? -- "We are masters in the following areas"

But I would prefer "We have expertise in following areas"

Reply:
defo "the following areas" but doesn't matter as much